CHANGELOG.md

```diff
@@ -27,6 +27,30 @@ Those changes in added, changed or breaking changes, should include usage exampl
 * Renamed `set_account_identifier` to `set_avs` [#365](https://github.com/Layr-Labs/eigensdk-rs/pull/365)
   * The underlying call was renamed in [the v1.1.1 eigenlayer-middleware release](https://github.com/Layr-Labs/eigenlayer-middleware/releases/tag/v1.1.1-testnet-slashing).
 
+* Changed the signature of `build_avs_registry_chain_writer` in [#384](https://github.com/Layr-Labs/eigensdk-rs/pull/384).
+  * The `operator_state_retriever_addr` parameter was removed.
+  * The `service_manager_addr` parameter was added.
```

Comment:
```suggestion
  * The `operator_state_retriever_addr` parameter was replaced by `service_manager_addr`.
```

Reply:
Let's also mention that this was done because later middleware versions don't have a `ServiceManager`.